normalize_license_text/normalize_class.py

```diff
@@ -78,7 +78,7 @@ def equivalentwords(self, normalized_string):
                     wordreplaced, wordtoreplace, normalized_string)
             equivalentfile.close()
             return normalized_string
-        except:
+        except (NameError):
             print("There was some problem with accessing the equivalent words file.")
```

Comment:
You could leave the `except:` as is, but replace the `print("` with `raise exception("` which will throw an exception to the next level up.  Otherwise a null string is returned and you get a null pointer exception which is harder to debug.

Reply:
Done the Changes.